cdap-docs/tools/doc-change.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#  Copyright © 2016 Cask Data, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Changes a doc set in place, modifying the headers.
+#
+# To change GitHub pages;
+# - Do it by changing a local copy of GH Pages, using the branch gh-pages-working:
+#
+# - Change all doc sets, declaring that '3.6.0' is the 'current' version:
+# ./doc-change.py ../../cdap-gh-pages/cdap --ghpages --current=3.6.0
+#
+# For testing, use:
+#
+# ./doc-change.py ../../cdap-gh-pages/cdap --ghpages --current=3.6.0 2.5.0
+#
+# ./doc-change.py ../../cdap-gh-pages/cdap --ghpages --current=3.6.0 3.6.0
+#
+# ./doc-change.py ../../cdap-gh-pages/cdap --ghpages --current=3.6.0 current
+#
+# To change doc server pages:
+# - Make a copy of the script on the web servers
+# - Replace the current symlink with a copy of the directory it points to, naming it as "current"
+#
+# - Change all doc sets, declaring that '3.6.0' is the 'current' version:
+# ./doc-change.py /var/www/docs/cdap --current=3.6.0
+#
+# For testing, use:
+#
+# ./doc-change.py /var/www/docs/cdap --current=3.6.0 2.5.0
+#
+# ./doc-change.py /var/www/docs/cdap --current=3.6.0 3.6.0
+#
+# ./doc-change.py /var/www/docs/cdap --current=3.6.0 current
+
+from optparse import OptionParser
+import os
+import sys
+
+def parse_options():
+    """ Parses args options.
+    """
+
+    parser = OptionParser(
+        usage="%prog [options] doc-directory [doc-set]",
```

Comment:
You show "prog options dir set" here, but use "prog dir options set" in all of your examples in the comments. Be consistent.